Dockerfile

```diff
@@ -34,6 +34,7 @@ ENV PHP_PM_MIN_SPARE_SERVERS 2
 ENV PHP_PM_MAX_SPARE_SERVERS 6
 ENV APP_MAGE_MODE default
 
+COPY lib/* /usr/local/etc/php/zend
```

Comment:
Add trailing slash to `/usr/local/etc/php/zend` for consistency with other `COPY` lines